cmd/notation/cert/cleanupTest.go

```diff
@@ -0,0 +1,116 @@
+package cert
+
+import (
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/notaryproject/notation-go/config"
+	"github.com/notaryproject/notation-go/dir"
+	"github.com/notaryproject/notation/cmd/notation/internal/truststore"
+	"github.com/notaryproject/notation/internal/osutil"
+	"github.com/spf13/cobra"
+)
+
+type certCleanupTestOpts struct {
+	keyName string
+}
+
+func certCleanupTestCommand(opts *certCleanupTestOpts) *cobra.Command {
+	if opts == nil {
+		opts = &certCleanupTestOpts{}
+	}
+	command := &cobra.Command{
+		Use:   "cleanup-test <common_name>",
+		Short: "Clean up a test key and its corresponding self-signed certificate created by the generate-test command.",
+		Args: func(cmd *cobra.Command, args []string) error {
+			if len(args) == 0 {
+				return errors.New("missing certificate common_name")
+			}
+			opts.keyName = args[0]
+			return nil
+		},
+		Long: `Clean up a test key and its corresponding self-signed certificate
+
+Example - Clean up a test key and its corresponding self-signed certificate named "wabbit-networks.io":
+  notation cert cleanup-test "wabbit-networks.io"
+`,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return cleanupTestCert(opts)
+		},
+	}
+	return command
+}
+
+func cleanupTestCert(opts *certCleanupTestOpts) error {
+	keyName := opts.keyName
+	var finalError []error
+	err := config.LoadExecSaveSigningKeys(func(keys *config.SigningKeys) error {
+		keySuite, err := keys.Get(keyName)
+		if err != nil {
+			return err
+		}
+		if keySuite.ExternalKey != nil {
+			return errors.New("cleanup-test can only apply to non-external keys created by the generate-test command")
+		}
+		if keySuite.X509KeyPair == nil {
+			return errors.New("cleanup-test requires key pair files created by the generate-test command")
+		}
+		// delete the key file from keyPath
+		keyPath := keySuite.X509KeyPair.KeyPath
+		err = osutil.DeleteFile(keyPath)
+		if err != nil {
+			if os.IsNotExist(err) {
+				fmt.Printf("Success: key file %q does not exist. It is already been cleaned up.\n", keyPath)
+			} else {
+				finalError = append(finalError, fmt.Errorf("cannot delete the key file %q during cleanup-test, %v", keyPath, err))
```

Comment:
nit: instead of having a `finalError` slice. Should we print the error directly?
```suggestion
				fmt.Fprintf(os.Stderr, "Failed: cannot delete the key file %q during cleanup-test: %v\n", keyPath, err))
```

Reply:
> nit: instead of having a `finalError` slice. Should we print the error directly?

@shizhMSFT I think the readability of the current printouts is higher. All succeeded cases are grouped at the top followed by all the failed cases grouped together as well. In this way, the printouts are very clear and easy to read. What do you think? 
/cc: @yizha1 @FeynmanZhou

Reply:
I think the output should reflect the order of the execution.